src/include/cache.i

```diff
@@ -312,7 +312,7 @@ __wt_eviction_dirty_needed(WT_SESSION_IMPL *session, u_int *pct_fullp)
  */
 static inline bool
 __wt_eviction_needed(
-    WT_SESSION_IMPL *session, bool busy, bool readonly, u_int *pct_fullp)
+    WT_SESSION_IMPL *session, bool busy, bool readonly, double *pct_fullp)
 {
 	WT_CACHE *cache;
 	u_int pct_dirty, pct_full;
```

Comment:
Why didn't you push the `double` change down into `__wt_eviction_clean_needed` and `__wt_eviction_dirty_needed` functions?